utilities/after_success.sh

```diff
@@ -32,7 +32,7 @@ if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" -a "${TRAVIS_BRANCH}" == "master" -
 
         cd ..
         utilities/update_docs_version.sh # Update version in READMEs
-        mvn deploy -DskipTests=true --settings target/travis/settings.xml -P sign-deploy
+        mvn clean deploy --settings ~/.m2/settings.xml -P sign-deploy
     else
         mvn deploy -DskipTests=true -Dgpg.skip=true --settings target/travis/settings.xml
```

Comment:
Ok, I agree that there is no need to re-run the tests for snapshots, though I think we should probably move `mvn cobertura:cobertura coveralls:report` to be done only in the non-snapshots branch (as it is only needed for the site).